phpmyfaq/admin/news.php

```diff
@@ -38,565 +39,113 @@
+    'defaultUrl' => $faqConfig->getDefaultUrl(),
+    'enableWysiwyg' => $faqConfig->get('main.enableWysiwygEditor'),
+    'ad_news_add' => Translation::get('ad_news_add'),
+    'csrfToken_saveNews' => Token::getInstance()->getTokenString('save-news'),
+    'ad_news_author_name' => Translation::get('ad_news_author_name'),
+    'ad_news_set_active' => Translation::get('ad_news_set_active'),
+    'ad_news_link_url' => Translation::get('ad_news_link_url'),
+    'ad_news_link_title' => Translation::get('ad_news_link_title'),
+    'ad_news_link_target' => Translation::get('ad_news_link_target'),
+    'ad_news_link_window' => Translation::get('ad_news_link_window'),
+    'ad_news_link_faq' => Translation::get('ad_news_link_faq'),
+    'ad_news_link_parent' => Translation::get('ad_news_link_parent'),
+    'selectLanguage' => LanguageHelper::renderSelectLanguage($faqLangCode, false, [], 'langTo'),
+    'ad_news_expiration_window' => Translation::get('ad_news_expiration_window'),
+    'ad_news_from' => Translation::get('ad_news_from'),
+    'ad_news_to' => Translation::get('ad_news_to'),
+    'ad_entry_back' => Translation::get('ad_entry_back'),
+    'ad_menu_news_add' => Translation::get('ad_menu_news_add'),
+    'ad_news_headline' => Translation::get('ad_news_headline'),
+    'ad_news_date' => Translation::get('ad_news_date'),
+    'ad_news_update' => Translation::get('ad_news_update'),
+    'ad_news_delete' => Translation::get('ad_news_delete'),
+    'ad_news_nodata' => Translation::get('ad_news_nodata'),
+    'ad_news_edit' => Translation::get('ad_news_edit'),
+    'ad_news_header' => Translation::get('ad_news_header'),
+    'ad_news_text' => Translation::get('ad_news_text'),
+    'ad_news_allowComments' => Translation::get('ad_news_allowComments'),
+    'ad_entry_locale' => Translation::get('ad_entry_locale'),
+    'ad_entry_comment' => Translation::get('ad_entry_comment'),
+    'ad_entry_commentby' => Translation::get('ad_entry_commentby'),
+    'newsCommentDate' => Translation::get('newsCommentDate'),
+    'ad_news_insertfail' => Translation::get('ad_news_insertfail'),
+    'msgNews' => Translation::get('msgNews'),
+    'ad_news_data' => Translation::get('ad_news_data'),
+    'ad_news_del' => Translation::get('ad_news_del'),
+    'ad_news_nodelete' => Translation::get('ad_news_nodelete'),
+    'ad_news_yesdelete' => Translation::get('ad_news_yesdelete'),
+    'ad_news_delsuc' => Translation::get('ad_news_delsuc'),
+    'ad_news_updatesuc' => Translation::get('ad_news_updatesuc'),
+    'msgDeleteNews' => Translation::get('msgDeleteNews'),
+    'csrfToken_deleteNews' => Token::getInstance()->getTokenString('delete-news'),
+    'csrfToken_updateNews' => Token::getInstance()->getTokenString('update-news'),
+    'ad_entry_active' => Translation::get('ad_entry_active'),
+    'csrfToken_activateNews' => Token::getInstance()->getTokenString('activate-news')
+];
+
+$filterCreateIsoDate = new TwigFilter('createIsoDate', function ($string) {
```

Comment:
We could move the 2 TwigerFilter classes to the Template folder and create new classes and files there.

Reply:
I'll do it after merging together with refactoring the user.php and change it in all files.